backend/corpora/lambdas/api/v1/dataset.py

```diff
@@ -48,3 +55,24 @@ def get_status(dataset_uuid: str, user: str):
     for remove in ["dataset", "created_at", "updated_at"]:
         status.pop(remove)
     return make_response(jsonify(status), 200)
+
+
+@db_session()
+def cancel_dataset_download(dataset_uuid: str, user: str):
```

Comment:
Follow the naming convention of the other APs
```suggestion
def delete_dataset(dataset_uuid: str, user: str):
    """Cancels an inprogress upload."""
```